src/main/java/appeng/me/storage/MEMonitorIInventory.java

```diff
@@ -103,7 +103,7 @@ public IAEItemStack injectItems( final IAEItemStack input, final Actionable type
 	}
 
 	@Override
-	public IAEItemStack extractItems( final IAEItemStack request, final Actionable type, final BaseActionSource src )
+	public IAEItemStack extractItems( final IAEItemStack request, final Actionable type, final IActionSource src )
 	{
 		ItemStack out = ItemStack.EMPTY;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "out". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1854)